change-developers-mindset.md

```diff
@@ -0,0 +1,49 @@
+## Change the developers mindset
+
+## Context
+
+* Top-down InnerSource support exists.
+* 3k population of developers.
+* Middle Management is not supporting InnerSource.
+* There's already a successful InnerSource group in the early stages.
+* Code visibility is product dependent.
+
+## Problem  
+
+* How do you change developer mindset? It's difficult to push developers to do things.
+* Developers are resisting the change, they are in their comfort zone and it's hard to get them to go out of it.
+* Developer organization's maturity is high, so people are used to being in some hierarchy/rules.
+* Developers are formed in agile, the shift from agile is difficult
+
+## Forces  
+
+* Managers are previous developers, so they like the way they were promoted and they want to proceed in similar ways.
+* Managers restrict what developers can spend time on.
+* Top-down approach to the InnerSource.
+* Different teams within the company have to decide to proceed with InnerSource.
+* No formal training.
+* Processes are not clear.
+
+## Sketch (optional)
+
+## Solution  
+
+* Show reward/recognition of the developer team that is engaged in InnerSource.
+* Formalize training.
+* Clarify more processes.
+* Give middle management specific objectives to make InnerSource successful.
+* Listen to manager complaints and fears and counter them.
+
```

Comment:
I would classify this as an Idea (Unproven solutions). The solutions do need more detail.